GestionareStructuraEchipaDeFormula1/src/gestionareStructuraEchipaDeFormula1/servicii/AlteServicii.java

```diff
@@ -0,0 +1,36 @@
+package gestionareStructuraEchipaDeFormula1.servicii;
+
+import gestionareStructuraEchipaDeFormula1.entitati.Fabrica;
+import gestionareStructuraEchipaDeFormula1.entitati.Magazin;
+import gestionareStructuraEchipaDeFormula1.entitati.Sponsor;
+
+import java.util.List;
+
+public class AlteServicii {
```

Comment:
deobicei in proiect este necesar sa denumim clasele foarte sugestiv in functie de logica continuta